<launch>
    <!-- Launch Gazebo and multi-UAV MAVROS SITL -->
    <include file="$(find offboard_py)/launch/gazebo_simulation.launch"/>

    <!-- Load experiment parameters from YAML file -->
    <rosparam file="$(find offboard_py)/config/experiment_params.yaml" command="load" />

    <!-- Target drone node (pursued drone) -->
    <!--node name="target_node" pkg="offboard_py" type="target.py" output="screen"/-->

    <!-- rqt_reconfigure node -->
    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>
</launch>

